moskito-frontend/src/components/Forms/FilterCampsiteForm.js

```diff
@@ -33,16 +42,52 @@ export default function FilterCampsiteForm({getCampsites}) {
         getCampsites(fields)
         history.push('/campsites')
     }
+
+    function handleClick(suggestion) {
+        const place = suggestion.address.postalCode + " " + suggestion.address.city
+        getGeocode(place)
+        .then(result => {
+            const geocodes = result.Response.View[0].Result[0].Location.DisplayPosition
+            setValues(
+                {
+                    ...fields,
+                    latitude: geocodes.Latitude,
+                    longitude: geocodes.Longitude,
+                    postalCode: place
+                }
+            )
+        })
+        .catch(error => console.log('error', error));
+        setSuggestions([])
+    }
+
+    function handleLocationChange(event) {
+        setValues({
+            ...fields,
+            [event.target.name]: event.target.value
+        })
+        fields.postalCode.length > 2 ?
+        autocomplete(fields.postalCode)
+        .then(results => setSuggestions(results.suggestions)) 
+        .catch(() =>
+          setSuggestions([
+            { description: 'Service nicht verfügbar', googlePlaceId: 'error' },
```

Comment:
What do you need the googlePlaceId for? ;-)

Reply:
;) must be a copy/paste error